src/card.css

```diff
@@ -0,0 +1,109 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+}
+
+.product-card__image {
+  display: flex;
+  justify-content: center;
+  height: 134px;
+  width: 160px;
+  margin-bottom: 40px;
+}
+
+.product-card__title {
+  display: flex;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  width: 166px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.product-card__kod {
+  display: flex;
+  align-self: flex-start;
+  font-size: 10px;
+  width: 160px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px ;
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.product-card__stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin-bottom: 24px;
+}
+
+.product-card__feedback-value {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.product-card__order {
+  display: flex;
+  flex-direction: column;
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  width: 100%;
+  line-height: 18px;
+}
+
+.product-card__price-price {
+  color: #616070;
+  font-size: 12px;
+}
+
+.product-card__price-sum {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+}
+
+.product-card__buy-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  width: 100%;
```

Comment:
that's redundant, this element will take up all of the available space anyways
```suggestion
```

Reply:
deleted